packaging/wheel/publish.sh

```diff
@@ -5,4 +5,35 @@ version=$(./genversion.sh --print-only)
 version=${version#v}
 echo $version > bindings/python/VERSION
 rm -r dist
-python3 setup.py sdist
+
+# Determine if wheel.bdist_wheel is available for wheel.bdist_wheel in setup.py
+python3 -c 'import wheel' &> /dev/null
+wheel_available=$?
+if [ "${wheel_available}" -ne "0" ]; then
+    python3 -m pip install wheel
+    res=$?
+fi
+unset wheel_available
+
+if [ "$res" -ne "0" ]; then
+    echo "ERROR: Unable to find wheel and unable to install wheel with '$(command -v python3) -m pip install wheel'."
+    echo "       Please ensure wheel is available to $(command -v python3) and try again."
+    exit 1
+fi
```

Comment:
`packaging/wheel/setup.py` contains

https://github.com/xrootd/xrootd/blob/eddbe1323a63a808d48c556606e96fea1d7c81bb/packaging/wheel/setup.py#L4

so ensure that `wheel` actually exists, as `python3-wheel` isn't a requirement in the RPM builds and shouldn't be unless it is called upon explicitly. As this is an sdist build, then there's no reason to not install the required tools from PyPI and so attempt to do so and warn if you can't.

Reply:
Yeah, I messed up there.